List-Comprehensions.ipynb

```diff
@@ -2,50 +2,83 @@
  "cells": [
```

Comment:
<p>we</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuefische/ds-meetups/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>